Transport/Stamp/GpsSenderOptionsStamp.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace PetitPress\GpsMessengerBundle\Transport\Stamp;
+
+use Symfony\Component\Messenger\Stamp\NonSendableStampInterface;
+
+/**
+ * @author Damien AVETTA-RAYMOND <damien.avetta@gmail.com>
+ */
+class GpsSenderOptionsStamp implements NonSendableStampInterface
+{
+    /**
+     * @var array<string, mixed>
+     */
+    private array $options;
+
+    /**
+     * @param array<string, mixed> $options
+     */
+    public function __construct(array $options) {
```

Comment:
Please, put it on a new line. https://www.php-fig.org/psr/psr-12/#44-methods-and-functions